lib/ios.dart

```diff
@@ -185,27 +185,23 @@ void _applyImageiOS({
     print('$imagePath could not be loaded.');
     exit(1);
   }
-  for (final template in list) {
-    _saveImageiOS(template: template, image: image, targetPath: targetPath);
-  }
-}
 
-/// Saves splash screen image to the project
-void _saveImageiOS({
-  required _IosLaunchImageTemplate template,
-  required Image image,
-  required String targetPath,
-}) {
-  final newFile = copyResize(
-    image,
-    width: image.width * template.pixelDensity ~/ 4,
-    height: image.height * template.pixelDensity ~/ 4,
-    interpolation: Interpolation.cubic,
-  );
+  await Future.wait(
+    list.map(
+      (e) => Isolate.run(() async {
+        final newFile = copyResize(
+          image,
+          width: image.width * e.pixelDensity ~/ 4,
+          height: image.height * e.pixelDensity ~/ 4,
+          interpolation: Interpolation.cubic,
+        );
 
-  final file = File(targetPath + template.fileName);
-  file.createSync(recursive: true);
-  file.writeAsBytesSync(encodePng(newFile));
+        final file = File(targetPath! + e.fileName);
```

Comment:
I don't think the bang operator `!` should be needed since line 179 assures `targetPath` is not `null`.

Reply:
The reason why this is needed now is because the value is used inside an new async Isolate block of code.
Not sure what else I could do here? Any suggestions?

Reply:
Makes sense.